python/ray/_private/external_storage.py

```diff
@@ -331,7 +331,11 @@ def restore_spilled_objects(
     def delete_spilled_objects(self, urls: List[str]):
         for url in urls:
             path = parse_url_with_offset(url.decode()).base_url
-            os.remove(path)
+            try:
```

Comment:
Can you update the ABC method definition this method should handle worker failure & retry case?